vacasa/connect/connect.py

```diff
@@ -732,6 +732,73 @@ def create_reservation(self,
 
         return self._post(url, json={'data': {'attributes': payload}}, headers=headers).json()
 
+    def cancel_reservation(self,
```

Comment:
can you call this `created_cancelled_reservation` or something?  Just want it to be clear that action here is not "cancel a reservation" in the normal sense, it's more "create a cancelled reservation for bookkeeping purposes"

Reply:
You're right. That is confusing.